<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="widget">
   <property name="minimumSize">
    <size>
     <width>1038</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QLabel" name="labelCapture">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>431</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>捕获图</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCamera">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="128">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>结果图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReadImage">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../ui资源库/picture_128px_1229366_easyicon.net.ico</normaloff>../../ui资源库/picture_128px_1229366_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCapture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../ui资源库/screenshot_128px_1223095_easyicon.net.ico</normaloff>../../ui资源库/screenshot_128px_1223095_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenCamera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../ui资源库/photo_camera_128px_1230607_easyicon.net.ico</normaloff>../../ui资源库/photo_camera_128px_1230607_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>300</number>
    </property>
    <property name="autoRepeatInterval">
     <number>100</number>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGray">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../ui资源库/folder_arrow_128px_1222876_easyicon.net.ico</normaloff>../../ui资源库/folder_arrow_128px_1222876_easyicon.net.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>打开摄像头</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>截取图片</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>信息显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="information_name">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>490</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>名字</string>
    </property>
   </widget>
   <widget class="QLabel" name="information_num">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>410</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文彩云</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>号码</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOpenCamera</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnOpenCamera_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReadImage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnReadImage_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCapture</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnCapture_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGray</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>btnGray_Clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>843</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>883</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnOpenCamera_Clicked()</slot>
  <slot>btnCapture_Clicked()</slot>
  <slot>btnReadImage_Clicked()</slot>
  <slot>btnThreshold_Clicked()</slot>
  <slot>btnGray_Clicked()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
